deploy/mongodb/templates/clusterdefinition.yaml

```diff
@@ -33,10 +33,10 @@ spec:
           keys:
             - mongodb.conf
           defaultMode: 256
-        - name: mongodb-metrics-config
-          templateRef: mongodb-metrics-config
+        - name: mongodb-metrics-config-new
+          templateRef: mongodb-metrics-config-new
           namespace: {{ .Release.Namespace }}
-          volumeName: mongodb-metrics-config
+          volumeName: mongodb-metrics-config-new
           defaultMode: 0777
```

Comment:
why does config contents execute permission? @sophon-zt should this attribute be removed from configSpecs?